frontend/src/bundles/account-settings/components/settings/user-profile/avatar-container.tsx

```diff
@@ -29,7 +29,11 @@ const AvatarContainer: React.FC = () => {
                 />
                 <div className={styles.btnContainer}>
                     <UploadAvatarButton handleFileChange={handleFileChange} />
-                    <DeleteAvatarButton deleteFile={deleteFIle} />
+                    {selectedFile ? (
+                        <DeleteAvatarButton deleteFile={deleteFIle} />
+                    ) : (
+                        ''
+                    )}
```

Comment:
I think its better to use selectedFile && `<DeleteAvatarButton deleteFile={deleteFIle} />`